src/main/java/wolox/training/models/Book.java

```diff
@@ -21,30 +24,39 @@
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private long id;
     /** Genre for book entity */
+    @NonNull
     @Column(name = "GENRE", nullable = false)
     private String genre;
     /** Author for book entity */
+    @NonNull
     @Column(name = "AUTHOR", nullable = false)
     private String author;
     /** Image for book entity */
+    @NonNull
     @Column(name = "IMAGE", nullable = false)
     private String image;
     /** Title for book entity */
+    @NonNull
     @Column(name = "TITLE", nullable = false)
     private String title;
     /** Subtitle for book entity */
+    @NonNull
     @Column(name = "SUBTITLE", nullable = false)
     private String subtitle;
     /** Publisher for book entity */
+    @NonNull
     @Column(name = "PUBLISHER", nullable = false)
     private String publisher;
     /** Year publish for book entity */
+    @NonNull
     @Column(name = "YEAR", nullable = false)
     private String year;
     /** Number pages for book entity */
+    @NonNull
     @Column(name = "PAGES", nullable = false)
     private String pages;
     /** Number ISBN for book entity */
+    @NonNull
     @Column(name = "ISBN", nullable = false)
     private String isbn;
 
```

Comment:
You can use @AllArgsConstructor annotation and delete this method

Reply:
It was fixed at last commit